src/creature_states.c

```diff
@@ -2003,7 +2003,7 @@ short creature_follow_leader(struct Thing *creatng)
         return 1;
     }
     int fails_amount = cctrl->field_307;
```

Comment:
looks like you've identified field_307 so might as well give it a name I guess

Reply:
Sure, but I never touched that bit. Named it now.

Reply:
well you had to figure out what it does when touching the code right? if they're only named once you actually touch the exact line and not just the area around it, we'll never get rid of all those unnamed ones

Reply:
In this case I had not looked at field_307 at all, did not see it used. But when you mentioned it I did and it was only used here, so naming is always nice.